src/ensembl/src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSelectors.ts

```diff
@@ -23,7 +23,8 @@ import { RootState } from 'src/store';
 import {
   TranscriptsStatePerGene,
   Filters,
-  SortingRule
+  SortingRule,
+  filterPanelOpen
```

Comment:
Please remember to remove this import

Reply:
done.

@msmanoj and @azangru can you do  another review and approve if its all good. Thanks